### DISCUSSION TITLE
"Generate texture with size" does not work.

### DISCUSSION THREAD

**Discussion by Yesay1996** [shun126/UE5-DungeonGeneratorDemo#41] [Q&A]
When I press "Generate texture with size", no texture appears in the "ProceduralTextures" folder, I am using Unreal Engine 5.3 version.

https://github.com/user-attachments/assets/bc4a3857-2d65-48a5-b262-7fcfe731c16c

**Comment by shun126** [comment]
I have confirmed the issue. I will fix it as soon as possible.

**Reply by Yesay1996** [reply]
Version 5.3, now the engine crashes when clicking on 'Generate texture with size'
5.3バージョン、「Generate texture with size」をクリックすると、エンジンがクラッシュします

**Reply by shun126** [reply]
I am truly sorry. It was working on our end, so we thought it was fixed. If possible, could you send us the log from when it crashed?

**Comment by Yesay1996** [comment]
The following is the error log：




LoginId:6e9db88e4c4b2140f44173b219b6e9c1
EpicAccountId:0334cfe30d37475a846f9036b054821a


Unhandled Exception: EXCEPTION_ACCESS_VIOLATION reading address 0x0000000000000000


VCRUNTIME140
UnrealEditor_DungeonGenerator!UDungeonMiniMapTexture::DuplicateTexture() [D:\build\U5M-Marketplace\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\DungeonGenerator\Source\DungeonGenerator\Private\MiniMap\DungeonMiniMapTexture.cpp:95]
UnrealEditor_DungeonGeneratorEditor!FDungeonGenerateEditorModule::SaveTexturePackage() [D:\build\U5M-Marketplace\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\DungeonGenerator\Source\DungeonGeneratorEditor\Private\DungeonGeneratorEditorModule.cpp:683]
UnrealEditor_DungeonGeneratorEditor!FDungeonGenerateEditorModule::SavePackages() [D:\build\U5M-Marketplace\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\DungeonGenerator\Source\DungeonGeneratorEditor\Private\DungeonGeneratorEditorModule.cpp:621]
UnrealEditor_DungeonGeneratorEditor!FDungeonGenerateEditorModule::OnClickedGenerateTextureWithSizeButton() [D:\build\U5M-Marketplace\Sync\LocalBuilds\PluginTemp\HostProject\Plugins\DungeonGenerator\Source\DungeonGeneratorEditor\Private\DungeonGeneratorEditorModule.cpp:547]
UnrealEditor_DungeonGeneratorEditor!TBaseRawMethodDelegateInstance<0,FDungeonGenerateEditorModule,FReply __cdecl(void),FDefaultDelegateUserPolicy&gt;::Execute() [D:\RocketSync\5.3.0-27405482+++UE5+Release-5.3\Working\Engine\Source\Runtime\Core\Public\Delegates\DelegateInstancesImpl.h:518]
UnrealEditor_Slate
UnrealEditor_Slate
UnrealEditor_Slate
UnrealEditor_Slate
UnrealEditor_Slate
UnrealEditor_Slate
UnrealEditor_Slate
UnrealEditor_ApplicationCore
UnrealEditor_ApplicationCore
UnrealEditor_ApplicationCore
UnrealEditor_ApplicationCore
user32
user32
UnrealEditor_ApplicationCore
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
UnrealEditor
kernel32
ntdll





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "shun126/UE5-DungeonGeneratorDemo"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年8月5日(星期一) 上午9:54
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [shun126/UE5-DungeonGeneratorDemo] "Generate texture with size" does not work. (Discussion #41)





 
I am truly sorry. It was working on our end, so we thought it was fixed. If possible, could you send us the log from when it crashed?

**Reply by shun126** [reply]
Thanks to your help, I was able to identify the cause. I will fix it in version 1.6.4.
